modules/ROOT/pages/api-led-develop.adoc

```diff
@@ -3,108 +3,107 @@ ifndef::env-site,env-github[]
 
 . In Studio, select *File > New > Mule Project*.
 +
 image:new-project-dialog.png[New project dialog]
 . In the dialog box, provide the following values:
-  ** *Project Name*: `hello-world`
+  ** *Project Name*: `hello-world-app`
   ** *Runtime*: If there is a choice, select the latest version of Mule. This is the Mule runtime engine that is hosted within Studio for testing.
   ** Select the *Import a published API* tab.
     ... Click the green plus sign, and then select *From Exchange*.
     ... Log in to your organization if needed.
     ... Type `hello-world` in the search box.
-    ... Click *hello-world* to select it.
-    ... Click *Add* to move it to the *Selected modules* list.
-    ... Click *Finish*.
-. Scroll down to verify that the checkbox *Scaffold flows from these API specifications* is selected. This ensures that Studio adds default error handling and other helpful items to your project.
-. Click *Finish*.
-. Studio creates an XML file, `hello-world.xml` and opens it in the canvas *Message Flow* view,
-displaying some of the basic elements you'll need for the API interface:
+    ... Click *hello-world-api* to select it.
+    ... Click *Add* to move it to the *Selected modules* list, and then click *Finish*.
+. Verify that the checkbox *Scaffold flows from these API specifications* is selected, and click *Finish*. 
++
+This ensures that Studio adds default error handling and other helpful items to your project
+
+Studio creates a `hello-world.xml` XML file and opens it in the canvas *Message Flow* view, showing the basic elements you'll need for the API interface:
 
 image:my-first-project.png[View in Studio after importing API specification with scaffolding]
 
-If you scroll down the canvas, you can see all the scaffolding that has been added to your API specification
-to create an API:
+If you scroll down the canvas, you can see all the scaffolding that has been added to your app:
 
 * HTTP Listener is ready to receive requests
 * APIkit Router routes messages
 * Error handlers for different error conditions
-* A console, *hello-world-console*, which we won't use in this tutorial
-* The GET request to endpoint `/greeting`, where we will do the bulk of our configuration.
+* A console, *hello-world-app-console*, which you won't use for this project
+* The GET request to endpoint `/greeting`, where you do the bulk of your configuration.
 
 Notice that there are three different views of the canvas:
 
 image:canvas-tabs.png[One canvas and links to all three views, 500]
 
-* *Message Flow* view makes it easy to drag and drop modules, connectors, and other assets and create the relationships between them.
-* *Global Elements* view makes it easy to specify global elements used in multiple projects.
-* *Configuration XML* view makes it easy to edit the XML of a project directly.
+* *Message Flow* makes it easy to drag modules, connectors, and other assets and create the relationships between them.
```

Comment:
Is there a way to switch up the "makes it easy" verbiage? 

Use *Message Flow* to...